websites/D/Diziroll/presence.ts

```diff
@@ -0,0 +1,79 @@
+const presence = new Presence({
+    clientId: "818550584994168934"
+});
+
+let stream: { duration: number; currentTime: number; paused: boolean };
+presence.on(
+  "iFrameData",
+  (data: { duration: number; currentTime: number; paused: boolean }) =>
+    (stream = data)
+);
+
+/**
+ * Get Timestamps
+ * @param {Number} videoTime Current video time seconds
+ * @param {Number} videoDuration Video duration seconds
+ */
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
+
```

Comment:
Use built-in function, `presence.getTimestamps()`

Reply:
So just remove this and use it on line 66th

Reply:
thanks